module/Application/src/Module.php

```diff
@@ -0,0 +1,27 @@
+<?php
+/**
+ * @link      http://github.com/zendframework/ZendSkeletonApplication for the canonical source repository
+ * @copyright Copyright (c) 2005-2016 Zend Technologies USA Inc. (http://www.zend.com)
+ * @license   http://framework.zend.com/license/new-bsd New BSD License
+ */
+
+namespace Application;
+
+use Zend\Mvc\ModuleRouteListener;
+use Zend\Mvc\MvcEvent;
+
+class Module
+{
+    const VERSION = '3.0.0dev';
```

Comment:
This is shameless advertising, but `ocramius/package-versions` deals with it, eventually :)